mission_control/test/test_fixed_rudder_behavior.py

```diff
@@ -39,9 +39,10 @@
 import rosnode
 import rospy
 import rostest
+from xml.etree import ElementTree
 from mission_control.msg import ReportExecuteMissionState
 from mission_control.msg import FixedRudder
-from load_execute_mission import LoadExecuteMission
+from mission_interface import *
```

Comment:
@patapufate nit: consider being explicit about importation (the style checker should've complained :thinking:)

Reply:
Changed. JFYI roslint pep8 didn't complain. (I have checked again)